src/components/UserGraphs.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import PieChart from "./Graphs/PieChart";
+import VBarChart from "./Graphs/VBarChart";
+import HBarChart from "./Graphs/HBarChart";
+
+export default function UserGraphs({ profileStats, repos }) {
+  const colors = ['#2980b9','#2ecc71','#e67e22','#e74c3c','#9b59b6'];
+  const colorHovers = ['#2980b9','#27ae60','#d35400','#c0392b','#8e44ad'];
```

Comment:
Just a suggestion for your next project. How bout putting the colors in a util/theme folder then in the theme folder you have like:

```
const THEME = {
  colors: {
      // color blue
      info: {
         100: '#.....',
         200: '#.....',
      },
      // green
      success: {
      },
  }
  // you can also do it for breakpoints or spacing and such
}
```

Basically we can have a primary and secondary color for the brand colors. Info - color blue, success/positive - green, danger/negative - red, warning - orange.

Then the numbers are the colors shade. Like how much light a color has. Most libraries contain this. Like in Bootstrap they use those terms. Or in Material UI wherein you can customize the theme colors.